CONTRIBUTING.md

```diff
@@ -1,9 +1,27 @@
-# User Guide
+# Contributing Guidelines
 
-Installation and usage instructions are available
-[here](https://github.com/rstudio/rsconnect-jupyter/tree/master/docs).
+This documentation covers common tasks releated to development.
 
-# Developing `rsconnect-jupyter`
+For installation and usage instructions see [README.md](./README.md).
+
+<!-- markdown-toc start - Don't edit this section. Run M-x markdown-toc-refresh-toc -->
+**Table of Contents**
+
+- [Contributing Guidelines](#contributing-guidelines)
+- [Development](#development)
+    - [Trying out notebooks](#trying-out-notebooks)
+    - [Seeing code changes](#seeing-code-changes)
+- [Packaging](#packaging)
+- [Versioning and Releases](#versioning-and-releases)
+    - [Versioning](#versioning)
+    - [Releases](#releases)
+        - [Releasing on Conda Forge](#releasing-on-conda-forge)
+        - [Adding yourself as a rsconnect-jupyter conda-forge maintainer](#adding-yourself-as-a-rsconnect-jupyter-conda-forge-maintainer)
+
+<!-- markdown-toc end -->
+
+
+# Development
 
 Need to run this after checkout and when modifying the docker images
```

Comment:
We dropped support for Python 2 a while ago, so Python 2 examples can be dropped.